syntax = "proto3";

package alyncoin;

import "block_protos.proto";
import "transaction_protos.proto";

message BlockchainProto {
    int32 chain_id = 1;
    repeated BlockProto blocks = 2;
    repeated TransactionProto pending_transactions = 3;
    int32 difficulty = 4;
    double block_reward = 5;
}
